subsys/usb/class/bluetooth.c

```diff
@@ -195,11 +195,11 @@ static uint16_t hci_pkt_get_len(struct net_buf *buf,
 		break;
 	}
 	case BT_BUF_ISO_OUT: {
-		struct bt_hci_iso_data_hdr *iso_hdr;
+		struct bt_hci_iso_hdr *iso_hdr;
 
 		hdr_len = sizeof(*iso_hdr);
-		iso_hdr = (struct bt_hci_iso_data_hdr *)data;
-		len = sys_le16_to_cpu(iso_hdr->slen) + hdr_len;
+		iso_hdr = (struct bt_hci_iso_hdr *)data;
+		len = sys_le16_to_cpu(bt_iso_hdr_len(iso_hdr->len)) + hdr_len;
```

Comment:
len + len?

Reply:
Technically `hdr_len` should probably have been `hdr_size`, but these variable are used for ACL and CMDs as well, so I don't see it fit to change the naming of those in this PR.